frontends/precinct-scanner/src/screens/scan_warning_screen.tsx

```diff
@@ -31,31 +32,37 @@ function OvervoteWarningScreen({
   overvotes,
 }: OvervoteWarningScreenProps): JSX.Element {
   const [confirmTabulate, setConfirmTabulate] = useState(false);
-  const contestNames = overvotes.map(
-    (overvote) =>
+  const contestNames = overvotes
+    .map((overvote) =>
       find(
         electionDefinition.election.contests,
         (contest) => contest.id === overvote.contestId
-      ).title
-  );
+      )
+    )
+    .reduce<AnyContest[]>(
+      (a, c) => (a.findIndex((o) => o.title === c.title) >= 0 ? a : [...a, c]),
+      []
+    )
+    .map((c) => c.title);
 
   return (
     <ScreenMainCenterChild infoBar={false}>
       <ExclamationTriangle />
       <CenteredLargeProse>
         <h1>Too Many Votes</h1>
         <Text small>
-          You voted for too many choices for {toSentence(contestNames)}. To fix
-          this, take your ballot and ask a poll worker for a new ballot.
+          There are too many votes marked for: {toSentence(contestNames)}.
+        </Text>
+        <Text small>
+          If you wish to correct this, return your ballot to a poll worker for a
+          new ballot.
         </Text>
         <p>
-          <Button primary large onPress={scanner.returnBallot}>
-            Return my ballot
-          </Button>
-        </p>
-        <p>
-          <Button small onPress={() => setConfirmTabulate(true)}>
-            Count my ballot
+          <Button primary onPress={scanner.returnBallot}>
+            Return Ballot
+          </Button>{' '}
+          <Button onPress={() => setConfirmTabulate(true)}>
+            Count Ballot As Is
```

Comment:
I think "as is" could be replaced by something more common that could communicate the same idea. Maybe "Count Ballot Anyway" or just "Count Ballot"

I'm having a hard time putting my finger on why "as is" feels a little weird to me

Reply:
I know that "As Is" seems a little weird, but is it clear?

I feel like it's not as awkward when said aloud. Maybe because of the way it looks when spelled?

I believe we do want to make the voter pause here, so I don't mind if "As Is" causes the voter to pause and make the correct selection here.

I'm also wondering if "Count" is the wrong word to use because the ballot will not be "counted" if there is an overvote. Perhaps the label should use "Accept" as in "Accept Ballot Anyway" or "Accept Ballot As Is"

Reply:
Might have to test this one on some voters to get to the bottom of it.

Re: the word "count" - I was thinking about this a bit when editing these screens. I opted for "Count" since we have similar language elsewhere (e.g. "ballots counted" and "ballot not counted", etc). So I wanted it to be consistent. I also think "count" is a very plain word - plainer than "accept."

That being said, I think the word that really comes to mind here to describe what's happening is "cast." I'm wondering if "cast" feels plain enough or if it's too jargon-y. We could potentially use "cast" to refer to ballots and "count" to refer to specific votes.

Reply:
I think "cast" should be find in the context of voting:
- A voter "casts" their ballot.
- The scanner "counts" ballots and votes.

I'm changing all to "Cast Ballot As Is" instead of "count" or "accept".